lib/delta.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var util = require('util');
+var deltaPrefix = '\u2206';
+var altDeltaPrefix = '\u0394';
+
+module.exports = {
+  deltaCounterName : function(metricName) {
+    if (module.exports.hasDeltaPrefix(metricName)) {
+      return metricName;
+    }
+    return deltaPrefix + metricName;
+  },
+
+  hasDeltaPrefix : function(metricName) {
+    if (metricName) {
+      return metricName.startsWith(deltaPrefix);
+    }
+  },
+
+  resetAndGetName : function(counter, prefix, name, value) {
+    counter.dec(value);
+    return util.format('%s.%s', deltaPrefix + prefix, name.substring(1));
+  }
+};
```

Comment:
I prefer the convention to put module.exports assignment stmt short and at file bottom. e.g.
```
function deltaCounterName (metricName) {

}

function hasDeltaPrefix (metricName) {

}

function resetAndGetName (counter, prefix, name, value) {

}

module.exports = {
  deltaCounterName,
  hasDeltaPrefix,
  resetAndGetName
}
```